client/script.js

```diff
@@ -4955,7 +4955,7 @@ $(function() {
               html,
               () => {
                 gNoPreventDefault = !gNoPreventDefault;
-                localStorage.noPreventDefault = noPreventDefault;
+                localStorage.noPreventDefault = gNoPreventDefault
```

Comment:
Semicolon. Doesn't matter much but would be great. I'll merge it if no one else has anything to note.
```suggestion
                localStorage.noPreventDefault = gNoPreventDefault;
```